app/assets/stylesheets/common/base/_topic-list.scss

```diff
@@ -334,7 +334,7 @@
 .loading .topic-list {
   border: 0;
   box-shadow: none;
-  tr {
+  .topic-list-item {
```

Comment:
Nitpicking again, but I would rather use `topic-list-row` because item and data can be more easily confused with each other.

Reply:
addressed here: https://github.com/discourse/discourse/pull/14956